javatests/arcs/core/entity/EntityBaseTest.kt

```diff
@@ -297,6 +298,23 @@ class EntityBaseTest {
     }
   }
 
+  @Test
+  fun singletonFields_duration() {
+    val int = ArcsDuration.ofMillis(99991234)
+    assertThat(entity.duration).isNull()
+    entity.duration = int
+    assertThat(entity.duration).isEqualTo(int)
+    assertThat(entity.duration).isInstanceOf(ArcsDuration::class.java)
```

Comment:
This assertion is not required, since `int` is already known to be of type `ArcsDuration`